src/core/common/Sidebar/index.js

```diff
@@ -44,29 +45,34 @@ const styles = theme => ({
  * @constructor
  */
 const CustomSidebar = props => {
-    const { classes, resources, isSidebarOpen, onMenuClick, location } = props;
+    const { classes, isSidebarOpen, onMenuClick, location } = props;
     const currentPathname = get(location, 'pathname', null);
     const pathNameArray = currentPathname.split('/');
     const currentList = '/' + pathNameArray[1];
-    const menuType = getMenuType(currentPathname);
-    if (isSidebarOpen && menuType === SHORT_MENU) {
-        return (
-            <SidebarWithShortMenu
-              classes={classes}
-              currentList={currentList}
-              onMenuClick={onMenuClick}
-            />
-        );
-    } else if (isSidebarOpen && menuType === FULL_MENU) {
-        return (
-            <SidebarWithFullMenu
-              classes={classes}
-              resources={resources}
-              currentList={currentList}
-              onMenuClick={onMenuClick}
-            />
-        );
+    const menuItems = getMenuItems(currentPathname);
+    if (!isSidebarOpen) {
+        return null;
     }
+    return (
+        <Sidebar className={classes.sidebarBlock}>
+            <div className={classes.menuBlock}>
+                {menuItems.map((item, key) => {
+                    return (
```

Comment:
no need for the `return`, you could earn one indentation

Reply:
Done